spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewriteDataFilesProcedure.java

```diff
@@ -269,7 +269,6 @@ private void insertData(int filesCount) {
   }
 
   private List<Object[]> currentData() {
-    spark.sql("select * from " + tableName + ".files").show(200, false);
```

Comment:
I added this for test, but forgot to revert in the final commit. Hence removed in this.